### ISSUE TITLE
Installing deps doesn't work when using Python 3.10

### ISSUE THREAD

**Issue by achou11** [alexnathanson/solar-protocol#26]
When running `python3 -m pip install -r requirements.txt` with Python 3.10.6, the following error occurs:

```sh
ERROR: Could not find a version that satisfies the requirement skia-python==87.1 (from versions: 87.4)
ERROR: No matching distribution found for skia-python==87.1
```

Looks like `skia-python` needs to be updated to https://github.com/kyamagu/skia-python/releases/tag/v87.4 in order to fix the issue. Doesn't seem like there should be any issues with doing so, so hopefully a straightforward fix

Alternatively, pin the Python version to < 3.10

**Comment by achou11** [comment]
Similar issue with numpy it seems, where upgrading to 1.21.X is needed (although that may be a riskier change that will break other deps). probably best to pin the python version instead given this

**Comment by achou11** [comment]
Decided to just run 3.9 instead for now, which seems to work as expected :)

**Comment by jedahan** [comment]
Resolved in v1.1
